pycode/memilio-epidata/memilio/epidata/transformMobilityData.py

```diff
@@ -0,0 +1,149 @@
+#############################################################################
+# Copyright (C) 2020-2021 German Aerospace Center (DLR-SC)
+#
+# Authors: Martin J. Kuehn
+#
+# Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#############################################################################
+import os
+import pandas as pd
+import numpy as np
+
+from memilio.epidata import getDataIntoPandasDataFrame as gd
+from memilio.epidata import defaultDict as dd
+from memilio.epidata import geoModificationGermany as geoger
+
+
+def getMobilityFromFile(directory, mobility_file):
+    """! Gets a mobility matrix that is written in a plain txt file
+    under the given directory into a pandas data frame.
+
+    @param directory Path to folder where data is read.
+    @param mobility_file Mobility matrix file which has to be updated.
+    @return Mobility matrix data frame.
+    """
+    mobility_matrix = pd.read_csv(
+        directory + mobility_file + '.txt', sep=' ', header=None)
+
+    return mobility_matrix
+
+
+def createFederalStatesMobility(directory, mobility_file, file_format=dd.defaultDict['file_format']):
```

Comment:
File_format removed.I'd like to not provide defaults for the other two since directory default is still to be discussed and also the name of the mobility file should be discussed in the near future.